src/haxelib/client/Main.hx

```diff
@@ -244,7 +244,10 @@ class Main {
 	}
 
 	function version() {
-		print(VERSION_LONG);
+		Sys.println('Haxelib\'s version: ${VERSION_LONG}');
```

Comment:
It's better not to change output format for no reason. Some tools or CIs may rely on it.